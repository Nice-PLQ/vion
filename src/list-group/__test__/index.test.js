import { mount } from '@vue/test-utils';
import VListGroup from '../list-group.vue';
import VList from '../../list/list.vue';

describe('ListGroup', () => {
  test('渲染', () => {
    const wrapper = mount({
      template: `
        <v-list-group :border="false">
          <v-list title="列表标题" help="辅助标题" desc="详细说明"></v-list>
          <v-list title="列表标题" help="辅助标题" desc="详细说明"></v-list>
        </v-list-group>
      `
    }, {
      global: {
        components: { VListGroup, VList }
      }
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
