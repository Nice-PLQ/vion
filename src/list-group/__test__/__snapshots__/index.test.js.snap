// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListGroup 渲染 1`] = `
<div
  data-v-app=""
>
  
  <!--v-if-->
  <div
    class="v-list-group v-list_border_none"
  >
    
    <div
      class="v-list v-list_multiple"
    >
      <div
        class="v-list__title-wrap"
      >
        
        
        <div
          class="v-list__title"
        >
          
          <div
            class="v-list__title-txt"
          >
            列表标题
          </div>
          
          
          <div
            class="v-list__desc-txt v-list__desc-txt_single"
          >
            详细说明
          </div>
          
        </div>
      </div>
      <div
        class="v-list__content"
      >
        
        <span
          class="v-list__help"
        >
          辅助标题
        </span>
        
        <!---->
        
        
      </div>
    </div>
    <div
      class="v-list v-list_multiple"
    >
      <div
        class="v-list__title-wrap"
      >
        
        
        <div
          class="v-list__title"
        >
          
          <div
            class="v-list__title-txt"
          >
            列表标题
          </div>
          
          
          <div
            class="v-list__desc-txt v-list__desc-txt_single"
          >
            详细说明
          </div>
          
        </div>
      </div>
      <div
        class="v-list__content"
      >
        
        <span
          class="v-list__help"
        >
          辅助标题
        </span>
        
        <!---->
        
        
      </div>
    </div>
    
  </div>
  <!--v-if-->
  
</div>
`;
